# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/lld/hotfix-no-macho.patch
# Copyright (C) 2022 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- lld-15.0.1.src/CMakeLists.txt.vanilla	2022-09-26 15:16:47.697000000 +0200
+++ lld-15.0.1.src/CMakeLists.txt	2022-09-26 15:16:52.630000000 +0200
@@ -256,7 +256,6 @@
 add_subdirectory(docs)
 add_subdirectory(COFF)
 add_subdirectory(ELF)
-add_subdirectory(MachO)
 add_subdirectory(MinGW)
 add_subdirectory(wasm)
 
--- lld-15.0.1.src/tools/lld/lld.cpp.vanilla	2022-09-26 15:15:42.481000000 +0200
+++ lld-15.0.1.src/tools/lld/lld.cpp	2022-09-26 15:17:09.901000000 +0200
@@ -151,8 +151,6 @@
       return elf::link;
     else if (f == WinLink)
       return coff::link;
-    else if (f == Darwin)
-      return macho::link;
     else if (f == Wasm)
       return lld::wasm::link;
     else
--- lld-15.0.1.src/tools/lld/CMakeLists.txt.vanilla	2022-09-26 15:15:30.697000000 +0200
+++ lld-15.0.1.src/tools/lld/CMakeLists.txt	2022-09-26 15:15:34.541000000 +0200
@@ -14,7 +14,6 @@
   lldCommon
   lldCOFF
   lldELF
-  lldMachO
   lldMinGW
   lldWasm
   )
